src/app/containers/FrontPageSection/index.jsx

```diff
@@ -9,6 +12,40 @@ import groupShape from '../../models/propTypes/frontPageGroup';
 import { storyItem } from '../../models/propTypes/storyItem';
 import idSanitiser from '../../lib/utilities/idSanitiser';
 
+const TopMargin = styled.div`
+  margin-top: ${GEL_SPACING};
+`;
+
+// TODO this is a bodge until desktop width top stories get merged.
+//  At that point, delete TopMarginSixHundred.
+const TopMarginSixHundred = styled.div`
+  @media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem) {
+    margin-top: ${GEL_SPACING};
+  }
+`;
+
+// eslint-disable-next-line react/prop-types
+const MarginWrapper = ({ addMargin, as, children }) => {
+  // Conditionally add a `margin-top` to the `children`, mixing in
+  // `as` if present.
+  if (addMargin) {
+    return <TopMargin as={as}>{children}</TopMargin>;
+  }
+  if (as) {
+    // TODO this is a bodge until desktop width top stories get merged.
+    //  At that point, delete the below return statement and uncomment the following.
+
+    // const AsComponent = as;
+    // return <AsComponent>{children}</AsComponent>;
+    return <TopMarginSixHundred as={as}>{children}</TopMarginSixHundred>;
+  }
+  // TODO this is a bodge until desktop width top stories get merged.
+  //  At that point, delete the below return statement and uncomment the following.
+
+  // return children;
+  return <TopMarginSixHundred>{children}</TopMarginSixHundred>;
```

Comment:
Do we need the if? Can't we have just one return with the `as` attribute which it will be set it we pass it via props?
```
return <TopMarginSixHundred as={as}>{children}</TopMarginSixHundred>;
```

Reply:
In the current bodged state, you can get away with it, yeah. But you need to follow the commented out code for how this component should behave if desktop width top stories already existed, which needs that check